### PR TITLE
Create usdaddselectmenu.md

### PR DESCRIPTION
**Pull request by Bumblebee-3**
totally weird lol

### CODE REVIEW

File: src/functions/usdaddselectmenu.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+>prototype : 'selectMenu',
+>code: `
+>$sendMessage[$nonEscape[$if[$interactionData[values[0]]==opt1;You selected Option 1;You selected option 2]];no]
+>`
+>})
+>bot.onInteractionCreate()
+>```
+### Available Data of the Select Menus
+
+`$interactionData[message.id]` => the message id to which the button has attached.
```
Comment (Neodevils) [inline src/functions/usdaddselectmenu.md]: button for select menu?

File: src/functions/usdaddselectmenu.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+`$interactionData[author.id]` => The person who clicked the button.
```
Comment (Neodevils) [inline src/functions/usdaddselectmenu.md]: Those are needs to be changed to menu though.

File: src/functions/usdaddselectmenu.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+`$interactionData[channel.id]` => The channel in which the button is located.
```
Comment (Neodevils) [inline src/functions/usdaddselectmenu.md]: And this one @Bumblebee-3

**Review by Neodevils — CHANGES_REQUESTED** [review]
Those are needs to be changed @Leref

File: src/functions/usdaddselectmenu.md (hunk 3 above)
Comment (Bumblebee-3) [reply src/functions/usdaddselectmenu.md]: Heyyy, Sorry, didnt notice that...

Comment (Bumblebee-3) [reply src/functions/usdaddselectmenu.md]: I just fixed it

**Comment by Leref** [comment]
Conflicts and already updated.

**Pull request closed without merging**